lib/std/zig/system.zig

```diff
@@ -811,7 +815,7 @@ pub const NativeTargetInfo = struct {
     }
 
     fn preadMin(file: fs.File, buf: []u8, offset: u64, min_read_len: usize) !usize {
-        var i: u64 = 0;
+        var i: usize = 0;
```

Comment:
What if I have a large (>4GB) ELF file?

Reply:
This fn is allowed to return short reads. `i` is just an offset, which is relative to `min_read_len`. `usize` was the correct type all along.